app/main.py

```diff
@@ -1,3 +1,52 @@
+import math
+
+
 class Vector:
     # write your code here
-    pass
+    def __init__(self, x: float, y: float):
+        self.x = round(x, 2)
+        self.y = round(y, 2)
+
+    def __add__(self, other):
+        if isinstance(other, Vector):
+            return Vector(self.x + other.x, self.y + other.y)
+        return Vector(self.x + other, self.y + other)
```

Comment:
$11-13
it's better to use one more instance for int:
`if isinstance(other, Vector):
            return Vector(self.x + other.x, self.y + other.y)
if isinstance(other, int)
        return Vector(self.x + other, self.y + other)`

Reply:
fix this